proposal/distributed-workloads.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Distributed Workloads and Workload Orchestration
+---
+
+**Keywords**
+CodeFlare, AppWrapper, MCAD, InstaScale
+
+**Introduction**
+Distributed Workloads is a simple, user-friendly abstraction for scaling, queuing and resource management of distributed AI/ML and Python workloads. It consists of three components: CodeFlare SDK, Multi-Cluster Application Dispatcher (MCAD) and InstaScale.
+
+[CodeFlare SDK](https://github.com/project-codeflare/codeflare-sdk) is an intuitive user-friendly Python library that allows the user to request, access and manage remote resources with specs of their choice such as the desired distributed compute framework (ie. Ray), worker count, CPU, GPU and memory. The SDK can request to scale up or down the desired resources (ie. Ray Cluster) by generating and interfacing with an AppWrapper, a MCAD CRD, that submits the job request with desired resources to MCAD. Although currently the AppWrapper supports only the Ray framework (that has a dependency on [KubeRay](https://github.com/ray-project/kuberay)), future work will focus on extending the AppWrapper to other distributed frameworks as well.
+
+[Multi-Cluster Application Dispatcher](https://github.com/IBM/multi-cluster-app-dispatcher) (MCAD) is a Kubernetes controller providing mechanisms for applications to manage batch jobs in a single or multi-cluster environment. It is capable of (i) providing an abstraction for wrapping all resources of the job or application and treating them holistically, (ii) queuing job or application creation requests and applying different queuing policies, e.g., First In First Out, Priority, and (iii) dispatching the job to one of multiple clusters, where a MCAD queuing agent runs, using configurable dispatch policies.
+
+[InstaScale](https://github.com/IBM/multi-cluster-app-dispatcher/tree/cluster-autoscale) is an on-demand resource scaling tool in the cloud. Within the resource limits set by the user, InstaScale can quickly scale up aggregated resources for distributed computing jobs, and just as easily release the resources once the job (ie. MCAD AppWrapper job) is complete. 
```

Comment:
Can you update the repo url to `project-codeflare` org and instascale repo?
https://github.com/project-codeflare/instascale